app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java

```diff
@@ -188,6 +188,9 @@ public void onCreateOptionsMenu(@NonNull final Menu menu,
         }
     }
 
+    /**
+     * fix github issue #3262: replace possible dirty URL with clean URL when sharing
+     */
```

Comment:
I don't think it is a good idea to add these javadocs here, I would just add a
```
// share the clean url instead of the dirty one
```
before every changed line

Reply:
I've reset the head and deleted the annotation.